hack/generate-admin-kubeconfig.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+hostname=$(cat ./hack/kind-values.yaml | grep externalHostname | cut -d" " -f2- | tr -d '"')
```

Comment:
Couldn't we use `yq` (https://github.com/mikefarah/yq) for this, rather than grepping through YAML?

Reply:
I don't mind changing it. My general "rule of thumb" is if tools needs to be installed and is not part of most linux OS - its should not be in the readme, if explicitly not specified that its pre-requisite. Makes it cleaner